src/main/java/com/graphhopper/converter/resources/ConverterResourceGisgraphy.java

```diff
@@ -0,0 +1,199 @@
+	private static final String FORMAT_PARAMETER = "format";
+	private static final String APIKEY_PARAMETER = "apikey";
+	// geocoding specid=fic
+	private static final String GEOCODING_LIMIT_PARAMETER = "limitnbresult";
+	// search
+	private static final String SEARCH_LIMIT_PARAMETER = "to";
+	private static final String SEARCH_QUERY_PARAMETER = "q";
+
+
+	private final String geocodingUrl;
+	private final String reverseGeocodingUrl;
+	private final String searchURL;
+	private final String apiKey;
+	private final Client jerseyClient;
+
+	public ConverterResourceGisgraphy(String geocodingUrl,
+			String reverseGeocodingUrl, String searchURL, String apiKey,
+			Client jerseyClient) {
+		this.geocodingUrl = geocodingUrl;
+		this.reverseGeocodingUrl = reverseGeocodingUrl;
+		this.searchURL = searchURL;
+		this.jerseyClient = jerseyClient;
+		this.apiKey = apiKey;
+	}
+
+	@GET
+	@Timed
+	public Response handle(@QueryParam("q") @DefaultValue("") String query,
+			@QueryParam("point") @DefaultValue("") String point,
+			@QueryParam("radius") @DefaultValue("") String radius,
+			@QueryParam("country") @DefaultValue("") String country,
+			@QueryParam("limit") @DefaultValue("5") Integer limit,
+			@QueryParam("reverse") @DefaultValue("false") boolean reverse,
+			@QueryParam("autocomplete") @DefaultValue("false") boolean autocomplete) {
+		limit = fixLimit(limit);
+		checkParameters(query, reverse, autocomplete, point);
+		
+		String lat = null;
+	    String lng = null;
+		if (point!=null && !point.isEmpty() && point.indexOf(",")>0){
+			String[] cords = point.split(",");
+         lat = cords[0];
+         lng = cords[1];
+		}
+		
+
+		WebTarget target;
+		if (reverse) {
+			target = buildReverseGeocodingTarget(query, lat, lng, radius,
+					country, limit);
+		} else if (autocomplete) {
+			target = buildAutocompleteTarget(query, lat, lng, radius, country,
+					limit);
+		} else {
+			target = buildGeocodingTarget(query, lat, lng, radius, country,
+					limit);
+		}
+
+		Response response = target.request().accept("application/json").get();
+		Status status = new Status(response.getStatus(), response
+				.getStatusInfo().getReasonPhrase());
+		failIfResponseNotSuccessful(target, status);
+
+
+		try {
+			if (!autocomplete) {
+				GisgraphyGeocodingResult feed = response
+						.readEntity(GisgraphyGeocodingResult.class);
+				return Converter.convertFromGisgraphyList(feed.result, status);
+			}
+			else {
+				GisgraphySearchResult feed = response
+						.readEntity(GisgraphySearchResult.class);
+				return Converter.convertFromGisgraphySearchList(feed.getResponse(), status);
+			}
+		} catch (Exception e) {
+			LOGGER.error("There was an issue with the target "
+					+ target.getUri() + " the provider returned: "
+					+ status.code + " - " + status.message);
+			throw new BadRequestException(
+					"error deserializing geocoding feed");
+		}
+	}
+
+	protected void processFeedException(WebTarget target, Status status) {
```

Comment:
This method is never called and does nothing? I think we should remove it.

Reply:
yes, removed.